source/documentation/using_the_api/getting_updates.md

```diff
@@ -6,106 +6,38 @@ You can regularly compare this value with new queries to a register to see if th
 
 ### Return all new entries since your last update
 
-You can use the [`GET /entries/` endpoint](#getentries) with the `start` and
-`limit` parameters to return specific entries to a register. 
+You can use the [`GET /entries/` endpoint](#getentries) to return specific entries to a register. This takes the parameters `start` and `limit`. 
```

Comment:
I would only mention the use of `start` when it's needed in order to start fetching entries from a custom starting point. And not mention at all `limit`.